go.mod

```diff
@@ -139,6 +139,7 @@ require (
 	github.com/spf13/afero v1.6.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/src-d/gcfg v1.4.0 // indirect
+	github.com/stretchr/objx v0.5.2 // indirect
```

Comment:
why didn't go.sum change when this was added?

Reply:
It seems to be already included in go.sum when updating to K8s 1.30, but no worries it's removed when not using mock.Mock